.gitmodules

```diff
@@ -4,7 +4,7 @@
 	branch = eip-4844
 [submodule "prysm/prysm"]
 	path = prysm/prysm
-	url = https://github.com/Inphi/prysm
+	url = https://github.com/prysmaticlabs/prysm
 	branch = eip-4844
```

Comment:
the upstream branch is `eip4844`.

Reply:
Updated, I pulled the latest change from that branch though